### ISSUE TITLE
Rewriting URLs for images hosted on Google to higher resolution versions

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-android#2413]
@oa-s can you help to write up and assign to James?

Related: https://github.com/AlphaWallet/alpha-wallet-ios/pull/3909

**Comment by JamesSmartCell** [comment]
OpenSea supply better resolutions for the collection symbol? Nice.

**Comment by hboon** [comment]
@JamesSmartCell not exactly, it's asking for a higher resolution version from Google for the images that host images. But looks like OpenSea has lots of them.

**Comment by JamesSmartCell** [comment]
I think I get it. Yeah I've noticed that and wondered how to improve it. EG the ENS logo thumbnail looks awful when blown up to the new size.
